src/components/misc/icon.tsx

```diff
@@ -1,31 +1,53 @@
+/* tslint:disable */
 import * as React from 'react';
+import * as Redux from 'react-redux';
+import {setSelectedIcon} from "../../redux/actions";
+import {Icon as IIcon, Store} from '../../redux/store-interfaces';
 import './misc.less';
 
-interface PropTypes { src: string, title: string, columnWidth: number, space: number  };
+interface PropTypes { icon:IIcon,  iconClass: string, iconContainerClass: string, iconShowDescription: boolean, iconColor: string, setSelectedIcon: typeof setSelectedIcon};
```

Comment:
Trenger dere å sende inn 2 style classer til React Componenten ? 
Tror dere kan komme unna med bare 1 klasse på containeren og style icon elementet inni. 

Da kan man også bare kalle parameteret `className` for å gjøre det likt med hvordan react bruker klasser :)